src/app/core/callendar/containers/calendar-container.component.ts

```diff
@@ -47,12 +48,18 @@ export class CalendarContainerComponent implements OnInit {
         this.store.dispatch(new SearchForJudgeWithHearings(query));
     }
 
-    public eventClick(session) {
-        this.dialog.open(DetailsDialogComponent, {
-            width: 'auto',
-            minWidth: 350,
-            data: new SessionDialogDetails(session),
-            hasBackdrop: false
-        });
+    public eventClick(eventId) {
+        if (eventId instanceof CustomEvent) {
+            return;
+        }
+        this.store.pipe(select(fromSessions.getSessionById(eventId)))
+            .subscribe(session => {
+                this.dialog.open(DetailsDialogComponent, {
+                    width: 'auto',
+                    minWidth: 350,
+                    data: new SessionDialogDetails(session),
+                    hasBackdrop: false
+                });
+            }).unsubscribe();
```

Comment:
yes I know that this unsubscribe is ugly, but allows to prevent suddenly showing dialog on each screen where sessions are used. If there is a better way I'm open for a fix after monday ;)